### PR TITLE
Add chase_provider_decision_at column to application_chices

### PR DESCRIPTION
**Pull request by C-gyorfi**
## Context

We want to send chaser emails to providers when an application has been submitted for 20 working days
The 20 working days rule part of the business rules because it's a counterpart to the 40-day limit, therefore it would make sense to add it to the schema just like `reject_by_default_at`

## Changes proposed in this pull request

This PR adds the `chase_provider_decision_at` column to the `application_chices` table

## Guidance to review

Do we need to backfill this data? I assume no API updates needed, is that right?

## Link to Trello card

https://trello.com/c/BTWWHYOj/836-email-%EF%B8%8F-a-decision-is-needed-on-an-application-because-it-has-been-submitted-for-20-working-days-to-provider

## Things to check
- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -151,7 +152,7 @@
     t.string "magic_link_token"
     t.datetime "magic_link_token_sent_at"
     t.boolean "hide_in_reporting", default: false, null: false
-    t.bigint "course_from_find_id"
+    t.integer "course_from_find_id"
```
Comment (stevehook) [inline db/schema.rb]: Is this intentional, guess not?

I think I had exactly the same thing happen once and ended up resetting my dev db. Maybe a postgres version thing.

Comment (C-gyorfi) [reply db/schema.rb]: I believe its a version thing, according to the migration files this should be `integer` but a previous migration changed https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/commit/8e3dcb555bfdc2fd9432de7744bd4ff8899b96f9

we discussed this before with @tijmenb and it should be harmless

Comment (C-gyorfi) [reply db/schema.rb]: What is your `psql --version` @davidgisbey ?

Comment (stevehook) [reply db/schema.rb]: I'm on 10.10 but I've not upgraded for ages

**Comment by C-gyorfi** [comment]
> I think we should use the ChaserSent feature that was added yesterday for this?

Yes, but regardless, I think we need to set a `chase provider` date when we send an application to the provider as suggested by @benilovj on this WIP PR https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1290

**Comment by tijmenb** [comment]
We'd have the date in the ChaserSent table. Not sure what comment you're referring to?

**Comment by C-gyorfi** [comment]
The date in the ChaserSent table is when the chaser sent and the `chase_provider_decision_at` field would be populated when we send the application to the provider(20 business days from that date)

**Pull request merged**